src/openseadragon.js

```diff
@@ -1403,6 +1405,26 @@ function OpenSeadragon( options ){
             CHROMEEDGE: 7
         },
 
+        /**
+         * Keep track of which {@link Viewer}s have been created.
+         * - Key: {@link Element} to which a Viewer is attached.
+         * - Value: {@link Viewer} of the element defined by the key.
+         * @static
+         * @type {Object}
+         */
+        // eslint-disable-next-line no-undef
+        _viewers: new Map(),
```

Comment:
This will be our first `Map`. I'm a little concerned about it, because we support IE11, and it looks like that browser doesn't have complete Map support (see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#browser_compatibility). That said, it does seem like a great solution here, and I think we're using just the most basic aspects of Map in this case, which should work in IE11.

Reply:
Cool that you like the approach :)

I don't think there is a compatibility issue with IE11, since I'm only using the following methods:

|Method|Compatibility (according to MDN)|
|:---|:---|
|`get`|🟢 Full|
|`set`|🟠 Partial|
|`delete`|🟢 Full|

Regarding `set`, it returns `undefined` instead of the `Map` object, which disallows chaining like this:

```js
myMap.set('bar', 'foo')
     .set(1, 'foobar')
     .set(2, 'baz');
```

However, I'm not making use of that construct

Reply:
Great, thank you for verifying this!